core/arch/arm/include/mm/core_mmu.h

```diff
@@ -115,6 +116,12 @@ struct core_mmu_phys_mem {
 		__used __section("phys_mem_map_section") = \
 		{ #addr, (type), (addr), (size) }
 
+#define register_sdp_mem(pa, sz) \
+	static const struct core_mmu_phys_mem __phys_sdp_mem_ ## pa\
```

Comment:
... and yes... i need this label to be able to register several SDP memories. It does not cost much.